purchase_order_analytic_search/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Purchase Order Analytic Search
+==============================
+
+Organizations often require to quickly find the purchase orders associated to
+an analytic account, searching by it's code, name or project/account manager.
+
+This module introduces the possibility to search purchase orders by analytic
+account.
+
+It also introduces a new menu entry in Purchasing to list purchase order lines.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Set 'Analytic Accounting' and 'Analytic Accounting for
```

Comment:
I would say that this fits better in the configuration section, doesn't it? :thinking: